chore: disable RabbitMQ service and related workflows

gabezurita opened the pull request:
This PR disables RabbitMQ service and related workflows as part of service shutdown https://github.com/department-of-veterans-affairs/abd-vro/issues/3858

Changes:
- Comment out RabbitMQ service in docker-compose.yml
- Remove RabbitMQ from image lists in scripts/image-names.sh and scripts/image_vars.src
- Comment out lib-rabbitmq-connector in settings.gradle
- Add documentation comments with issue reference #3742

Documentation:
- Disabling issue: https://github.com/department-of-veterans-affairs/abd-vro/issues/3742
- Re-enable conditions: When RabbitMQ messaging services are needed again in the future

gabezurita committed:
chore: disable RabbitMQ service and related workflows

This commit disables RabbitMQ service and related workflows as part of service shutdown.

Changes:
- Comment out RabbitMQ service in docker-compose.yml
- Remove RabbitMQ from image lists in scripts/image-names.sh and scripts/image_vars.src
- Comment out lib-rabbitmq-connector in settings.gradle
- Add documentation comments with issue reference #3742

Documentation:
- Disabling issue: https://github.com/department-of-veterans-affairs/abd-vro/issues/3742
- Re-enable conditions: When RabbitMQ messaging services are needed again in the future

dfitchett commented on .github/dependabot.yml:
I know not a huge deal, but could we make all the indents the same for commented out sections? I messed it up with the bgs section 😅 . Only including this since I already requested a different change.

gabezurita replied:
Done, see: https://github.com/department-of-veterans-affairs/abd-vro/pull/4015/commits/3efdb008ffd9720b135c5d34e23700f365085b04

dfitchett commented on settings.gradle:
So we can still run gradlew docker commands, you'll also need to comment out the `:domain-xample:xample-api-controller` because it depends on the `:shared:lib-rabbit-connector`. This should also fix broken CI stuff.

gabezurita replied:
Ah, yep! I'll fix it. Sorry for moving this to code review without a green CI!

Done, see: https://github.com/department-of-veterans-affairs/abd-vro/pull/4015/commits/3efdb008ffd9720b135c5d34e23700f365085b04

gabezurita replied:
Done, see: https://github.com/department-of-veterans-affairs/abd-vro/pull/4015/commits/3efdb008ffd9720b135c5d34e23700f365085b04

dfitchett reviewed (changes requested):
One small change, one nit!

gabezurita committed:
chore: disable RabbitMQ dependent bits
'https://github.com/department-of-veterans-affairs/abd-vro/pull/4015#discussion_r1936397974'

gabezurita commented:
Moving this back to ready for code review to get a full CI run.

gabezurita commented:
@dfitchett, I believe I've fixed the CI and existing-indent issue. This is now _actually_ ready for review!

dfitchett reviewed (approved):
LGTM! 🚀

gabezurita committed:
Merge branch 'develop' into gabezurita/shut-down-rabbit-mq
